charts/local-kubevirt/update.bash

```diff
@@ -63,7 +63,7 @@ yq -s '.kind + "-" + .metadata.name + ".yaml"' ./cdi-operator.yaml
 rm ./cdi-operator.yaml
 
 for f in *; do 
-    l=$(echo "$f" | tr '[A-Z]' '[a-z]')
+    l=$(echo "$f" | tr '[A-Z:]' '[a-z-]')
```

Comment:
just highlighting, this ensure the chart template file names comply with https://github.com/kubermatic/kubermatic/pull/12330 even in the future.